pkg/secrets/flags.go

```diff
@@ -37,35 +29,19 @@ func (o *CLIOptions) Bind(fs *flag.FlagSet, getenv func(string) string, censor *
 	}
 }
 
-func (o *CLIOptions) Validate() []error {
-	var errs []error
-	var credentialsProviderConfigured []string
-	if (o.BwUser == "") != (o.BwPasswordPath == "") {
-		errs = append(errs, fmt.Errorf("--bw-user and --bw-password-path must be specified together"))
-	} else if o.BwUser != "" {
-		credentialsProviderConfigured = append(credentialsProviderConfigured, "bitwarden")
-	}
+func (o *CLIOptions) Validate() error {
 	if o.VaultAddr != "" || o.VaultToken != "" || o.VaultTokenFile != "" || o.VaultPrefix != "" {
```

Comment:
We can remove this condition, vault must be configured now and the inner `if` already checks the completeness of that

Reply:
addressed.